crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -1513,12 +1459,6 @@ mod private {
             for event in events {
                 self.maybe_apply_new_redaction(&event).await?; // TODO: Handle redaction for search index
```

Comment:
Let's also remove the `TODO` here; it's going to go to the search indexing task instead.